hack/app-sre/kustomization.yaml

```diff
@@ -0,0 +1,46 @@
+# This kustomize configuration deploys what is required for the hive-operator
+# *without* OLM.
+# Except:
+# It does not contain a HiveConfig.
+
+# Adds namespace to all resources.
+namespace: hive
+
+# The CRD resources are synced automatically by the `build-app-sre-template` make target.
+# The other resources should be curated manually.
+resources:
+- ../../config/namespace.yaml
+- ../../config/operator/operator_role.yaml
+- ../../config/operator/operator_role_binding.yaml
+- ../../config/operator/operator_deployment.yaml
+- ../../config/crds/hiveinternal.openshift.io_clustersyncleases.yaml
+- ../../config/crds/hiveinternal.openshift.io_clustersyncs.yaml
+- ../../config/crds/hiveinternal.openshift.io_fakeclusterinstalls.yaml
+- ../../config/crds/hive.openshift.io_checkpoints.yaml
+- ../../config/crds/hive.openshift.io_clusterclaims.yaml
+- ../../config/crds/hive.openshift.io_clusterdeployments.yaml
+- ../../config/crds/hive.openshift.io_clusterdeprovisions.yaml
+- ../../config/crds/hive.openshift.io_clusterimagesets.yaml
+- ../../config/crds/hive.openshift.io_clusterpools.yaml
+- ../../config/crds/hive.openshift.io_clusterprovisions.yaml
+- ../../config/crds/hive.openshift.io_clusterrelocates.yaml
+- ../../config/crds/hive.openshift.io_clusterstates.yaml
+- ../../config/crds/hive.openshift.io_dnszones.yaml
+- ../../config/crds/hive.openshift.io_hiveconfigs.yaml
+- ../../config/crds/hive.openshift.io_machinepoolnameleases.yaml
+- ../../config/crds/hive.openshift.io_machinepools.yaml
+- ../../config/crds/hive.openshift.io_selectorsyncidentityproviders.yaml
+- ../../config/crds/hive.openshift.io_selectorsyncsets.yaml
+- ../../config/crds/hive.openshift.io_syncidentityproviders.yaml
+- ../../config/crds/hive.openshift.io_syncsets.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+# Use app-sre-supplied variables to pull, by digest, the image for the current commit
+images:
+- digest: ${IMAGE_DIGEST}
```

Comment:
Alternative: use the tag and don't worry about Disappearing Digest Syndrome.

Reply:
i think we should go as conventional as possible and deploy this with IMAGE_TAG?

Reply:
Yeah, probably. And cut over to by-digest separately. Principle of least change.

Reply:
Done.